packages/changed-elements-react/src/widgets/V2Widget/componets/VersionList.tsx

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { IModelApp } from "@itwin/core-frontend";
+import { ReactElement } from "react";
+import { VersionListEntry } from "./VersionEntries";
+import { VersionState } from "../models/VersionState";
+import { NamedVersion } from "../../../clients/iModelsClient";
+import "./styles/VersionCompareSelectWidget.scss";
+
+interface VersionListProps {
+  entries: VersionState[];
+  currentVersion: VersionState;
+  selectedVersionChangesetId: string | undefined;
+  onVersionClicked: (targetVersion: NamedVersion) => void;
+}
+
+/**
+ * Component that displays named versions (non current).
+ */
+export function VersionList(props: VersionListProps): ReactElement {
+  return (
+    <div className="version-compare-row version-compare-list">
+      <div className="version-container-table">
+        <div className="version-container-header" style={{ borderBottom:"solid 2px black"}}>
```

Comment:
How come this style is defined here and not a part of the definition in the scss file?